[docs] Cloud Select database version (#11092)
yugabyte/yugabyte-db: docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters.md

{{< warning title="Important" >}}

> Save your database credentials in a safe place. If you lose these credentials you will not be able to use the database.

Reviewer: ```suggestion
Save your database credentials in a safe place. If you lose these credentials, you won't be able to use the database.
```

{{< /warning >}}

## Viewing the cluster

> After you complete the wizard, the **Clusters** page appears with the provisioning of your new cluster in progress.

Reviewer: ```suggestion
After you complete the wizard, the **Clusters** page appears, showing the provisioning of your new cluster in progress.
```
